src/store/actions.ts

```diff
@@ -157,3 +158,33 @@ export const initWallet = () => {
         }
     };
 };
+
+export const cancelOrder = (order: SignedOrder) => {
+    return async (dispatch: any, getState: any) => {
+        const state = getState();
+        const ethAccount = getEthAccount(state);
+
+        const web3Wrapper = await getWeb3WrapperOrThrow();
+        const networkId = await web3Wrapper.getNetworkIdAsync();
+        const contractWrappers = new ContractWrappers(web3Wrapper.getProvider(), { networkId });
+        const tx = await contractWrappers.exchange.cancelOrderAsync(order, {
+            gasLimit: 1000000,
```

Comment:
@fvictorio I can move the repeated options object to `/src/constants`. Also, the options may be _optional_ (setup via an env variable) because I'm not 100% sure if the `out of gas` error has to do with using ganache. Have to test this on a test net.

Reply:
Yes, let's do that.